guides/source/form_helpers.md

```diff
@@ -1135,7 +1285,26 @@ As a convenience you can instead pass the symbol `:all_blank` which will create
 
 ### Adding Fields on the Fly
 
-Rather than rendering multiple sets of fields ahead of time you may wish to add them only when a user clicks on an "Add new address" button. Rails does not provide any built-in support for this. When generating new sets of fields you must ensure the key of the associated array is unique - the current JavaScript date (milliseconds since the [epoch](https://en.wikipedia.org/wiki/Unix_time)) is a common choice.
+Rather than rendering multiple sets of fields ahead of time you may wish to add them only when a user clicks on an "Add new address" button. Rails does not provide any built-in support for this though.
```

Comment:
Haha, yeah agree, it's a bit "good luck with that".  This section can be removed IMO, not adding any value. (The advice would involve talking about turbo and stimulus as far as how I'd do this...there might be other ways or external gems...but I think all of that is out of scope for this guide).

Reply:
>Should this have been removed? "When generating new sets of fields you must ensure the key of the associated array is unique - the current JavaScript date (milliseconds since the [epoch](https://en.wikipedia.org/wiki/Unix_time)) is a common choice."

The 2nd paragraph, I'm not seeing which line that is referring to?